code/game/dna/dna_modifier.dm

```diff
@@ -972,6 +974,7 @@
 				databuf.dna = src.connected.occupant.dna.Clone()
 				if(ishuman(connected.occupant))
 					databuf.dna.real_name=connected.occupant.name
+					databuf.dna.flavor_text=connected.occupant.flavor_text
```

Comment:
Hm, I removed line 977 from dna_modifer.dm, so I don't know why this isn't listed as outdated.